### PR TITLE
feat: update button styles

### LINKED ISSUE #8577: ENA-231: Make setting switches colorblind friendly
### The problem
On setting switches that use buttons with icons, such as the shown/hidden switch or the rotation style setting, each icon's active state is colored in, and each icon's inactive state is grayed out. But looking at it in grayscale, you actually can't tell which one is on. See the following image:
Image of the shown/hidden switch in color compared to an image of it in grayscale
For colorblind users, this can be an issue.

### The solution
We shouldn't rely on colored-in versus black-and-white to show which option was selected. Instead, we could change the background color when an option is selected, and make it white, not gray, when the option isn't selected, like so:
The Show button has a blue background, and the Hide button has a white background.
This difference is easier for colorblind users to see.

### PR DESCRIPTION
**Pull request by aoneill01**
Link to Figma specs in the Jira ticket.

### Resolves

- Resolves [ENA-280](https://scratchfoundation.atlassian.net/browse/ENA-280)
- Resolves #8577 

### Proposed Changes

- Create a `<ToggleButtons />` component for reusable logic/styling of show/hide, stage size, and rotation mode buttons
- Update styling for fullscreen and sound undo/redo buttons

Screenshot 2023-04-18 at 5 39 23 PM

### Reason for Changes

This is part of the high contrast color theme epic. The purple accent color made toggle buttons (show/hide, stage size, direction mode) harder to detect selection. An updated design improves accessibility and increases consistency across various buttons.

### Test Coverage

- Add unit tests for `<ToggleButtons />`
- Update existing integration tests for stage size

### Browser Coverage
Check the OS/browser combinations tested (At least 2)

Mac
 * [x] Chrome 
 * [x] Firefox 
 * [x] Safari
 
Windows
 * [x] Chrome 
 * [ ] Firefox 
 * [x] Edge
 
Chromebook
 * [ ] Chrome
 
iPad
* [ ] Safari

Android Tablet
* [ ] Chrome

[ENA-280]: https://scratchfoundation.atlassian.net/browse/ENA-280?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mxmou** [comment]
> Resolves [ENA-280](https://scratchfoundation.atlassian.net/browse/ENA-280)

I think this also fixes #8577.

[ENA-280]: https://scratchfoundation.atlassian.net/browse/ENA-280?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

File: src/components/toggle-buttons/toggle-buttons.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import styles from './toggle-buttons.css';
+
+const ToggleButtons = ({buttons, className, disabled}) => (
```
Comment (delasare) [inline src/components/toggle-buttons/toggle-buttons.jsx:7]: Love it!

**Review by delasare — APPROVED** [review]
:shipit:

**Pull request merged**